CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Github workflow for changelog verification ([#239](https://github.com/opensearch-project/opensearch-java/pull/239))
 - Github workflow for dependabot PRs ([#247](https://github.com/opensearch-project/opensearch-java/pull/247))
 - Add javadoc link for the client ([#255](https://github.com/opensearch-project/opensearch-java/pull/255))
+- Get rid of the dependency on OpenSearch core ([#281](https://github.com/opensearch-project/opensearch-java/pull/281))
```

Comment:
Sorry, I'm being really picky about the wording here but I think "get rid of the dependency..." is misleading. This commit introduces an option to avoid _using_ any dependency on OpenSearch core, but the dependency is still there.

Reply:
Oh sorry, I should have updated that, thx!